BricksAndHearts.UnitTests/ControllerTests/Admin/AdminControllerTests.cs

```diff
@@ -24,4 +31,37 @@ public void Index_WhenCalledByAnonymousUser_ReturnsViewWithLoginLink()
         result!.ViewData.Model.Should().BeOfType<AdminViewModel>()
             .Which.CurrentUser.Should().BeNull();
     }
+
+    [Fact]
+    public void GetAdminList_ReturnsViewWith_AdminListModel()
+    {
+        // Arrange
+        var fakeAdminService = A.Fake<IAdminService>();

```

Comment:
There's already a fake admin service in `AdminControllerTestsBase`, which this class inherits from. You should use that rather than creating a new one here